crates/rspack/tests/tree-shaking/array-side-effects/expected/main.js

```diff
@@ -1,46 +1,34 @@
 (self['webpackChunkwebpack'] = self['webpackChunkwebpack'] || []).push([["main"], {
 "./app.js": function (module, exports, __webpack_require__) {
-"use strict";
-Object.defineProperty(exports, "__esModule", {
-    value: true
-});
-Object.defineProperty(exports, "app", {
-    enumerable: true,
-    get: function() {
-        return app;
-    }
-});
-var _lib = __webpack_require__("./lib.js");
-function app() {}
-app.prototype.result = _lib.result;
+'use strict';
```

Comment:
The tree shaking test has test script for diff, it can be reviewed by @IWANABETHATGUY, others could skip it.